src/cart/crt_corpc.c

```diff
@@ -921,15 +924,19 @@ crt_corpc_req_hdlr(struct crt_rpc_priv *rpc_priv)
 
 		child_rpc_priv = container_of(child_rpc, struct crt_rpc_priv,
 					      crp_pub);
+
 		corpc_add_child_rpc(rpc_priv, child_rpc_priv);
 
 		child_rpc_priv->crp_grp_priv = co_info->co_grp_priv;
 
+		RPC_ADDREF(rpc_priv);
```

Comment:
just curious why need this addref and the corresponding decref in crt_corpc_reply_hdlr.
As in crt_req_send we already take ref for the parent RPC.

Reply:
This isnt strictly required anymore, but was needed if we previously returned error from corpc handling (instead of overriding rc). In that case if we returned error, the original rpc would get decreffed to 0 on exit.

Going forward i want to change how we are handling rpc add/decref and follow a model previously outlined by LiWei for iv ref counts -- to follow compiler like approach, where assignment of ptrs would follow with corresponding addref instead of relying on logic-based add/dec ref counting